InvenTree/InvenTree/mixins.py

```diff
@@ -46,17 +71,21 @@ def clean_data(self, data: dict) -> dict:
             data (dict): Data that should be sanatized.
 
         Returns:
-            dict: Profided data sanatized; still in the same order.
+            dict: Provided data sanatized; still in the same order.
         """
+
         clean_data = {}
+
         for k, v in data.items():
             if isinstance(v, str):
-                ret = clean(v)
```

Comment:
Suggestion: From a readability standpoint I would prefer to run `clean` here and then run a renamed `clean_string` with the output. Namesuggestion: clean_escape